tsconfig.json

```diff
@@ -26,14 +26,9 @@
     "test",
     "release",
     "app/main.prod.js",
-    "app/main.prod.js.map",
```

Comment:
I think `app/main.prod.js.map` is used

Reply:
don't believe so, typescript doesn't care about files other than .ts, .d.ts, .tsx, .js, .jsx(https://github.com/microsoft/TypeScript/issues/10939)

The # of files linted is the same with the old and new tsconfig
![image](https://user-images.githubusercontent.com/15751908/84857948-01dda880-b01f-11ea-8a4c-b42d647a78b3.png)

Reply:
Ahh I see, good point. Thanks for clarifying!